### ISSUE TITLE
Accessibility: Color contrast issues

### ISSUE THREAD

**Issue by josebui** [techmatters/terraso-web-client#234]
- All pages are showing contrast issues using: https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd

**Comment by Hanal-code** [comment]
@josebui  Change the background color of the avatar to #767676 please.

**Comment by Hanal-code** [comment]
For the main navigation items's hover and focus state, set the background-color to #EDEDED and the text color to #555555. 
Remove the focus state animation(circle) and use the same styling of the hover state(background color #EDEDED).  This will lighten up the hover state background color and darken the text color.
[mockup]
<img width="333" alt="image" src="https://user-images.githubusercontent.com/92342777/154368048-38e1eda7-9618-4d91-99d7-7c6a1c6d3d05.png">

**Comment by Hanal-code** [comment]
@jose I also think `<span class="MuiTouchRipple-root css-w0pj6f"></span>` is adding dark circle animation when being focused(tab key), so it triggers the color contrast alert as well. Can we remove that?

**Comment by josebui** [comment]
@Hanal-code The issue with that is that this means hardcoding values, is there a way to append the new value colors to the current palette? Maybe we don't need to change one but maybe we can add more there to track them

**Comment by josebui** [comment]
Hey @Hanal-code from what I am checking, the issues of color contrast are not related just to the hover or focus states, it seems that your solutions for 
```
For the main navigation items's hover and focus state, set the background-color to #EDEDED and the text color to #555555.
Remove the focus state animation(circle) and use the same styling of the hover state(background color #EDEDED). This will lighten up the hover state background color and darken the text color.
```
Are just talking about those states, when the main issue is the conlor contrast without those states. Also these color too should be part of the `Terraso Color Palette` in some form to avoid hardcoding values. Also I would suggest (at least for now) to avoid changing how Mui works to achieve accessibility, like disabling animations, or ripple effects

**Comment by Hanal-code** [comment]
@josebui  AXE DevTools seemed to find more color contrast issue than any other automated tools. Was it set to WCAG AAA? 
I doubled checked the main navigation link color:
<img width="464" alt="image" src="https://user-images.githubusercontent.com/92342777/155182727-8ae0c359-66ea-4296-92b0-10843f43cfc3.png">

the hyperlink color (#307F9C) against the white background and it passed the WCAG AA's 4.5:1 ratio requirement in the CCA tool.
<img width="442" alt="image" src="https://user-images.githubusercontent.com/92342777/155181616-aa72214a-a717-4710-8028-dda9475d9dd3.png">

It's puzzling why the different automated a11y evaluation tools gave us different results.
